uk_bin_collection/uk_bin_collection/councils/HerefordshireCouncil.py

```
@@ -30,35 +30,34 @@ def parse_data(self, page: str, **kwargs) -> dict:
         soup = BeautifulSoup(page.text, "html.parser")
         soup.prettify
 
-        data = {"bins": []}
-
-        checkValid = soup.find("p", id="selectedAddressResult")
+        checkValid = any("Your next collection days" in h2.get_text() for h2 in soup.find_all("h2"))
         if checkValid is None:
             raise ValueError("Address/UPRN not found")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)